sources/oceania/au/act/ACTmapi-Imagery202402.geojson

```diff
@@ -0,0 +1,195 @@
+{
+    "type": "Feature",
+    "properties": {
+        "id": "ACT202402",
+        "attribution": {
+            "url": "http://actmapi.act.gov.au/terms.html",
+            "text": "Aerial Imagery from ACTMapi ©ACT Government and MetroMap",
+            "required": true
+        },
+        "license_url": "https://osmlab.github.io/editor-layer-index/sources/oceania/au/act/ACTmapi-Imagery.PDF",
+        "privacy_policy_url": "https://www.esri.com/en-us/privacy/overview",
+        "name": "ACTmapi Imagery Feb 2024",
+        "url": "https://tiles.arcgis.com/tiles/E5n4f1VY84i0xSjy/arcgis/rest/services/ACT_Aerial_Imagery_202402_AGOL_Compatible/MapServer/tile/{zoom}/{y}/{x}",
+        "best": true,
+        "start_date": "2024-02",
+        "end_date": "2024-02",
+        "max_zoom": 21,
+        "country_code": "AU",
+        "type": "tms",
+        "icon": "https://actmapi.act.gov.au/img/apple-touch-icon.png",
+        "category": "photo",
+        "min_zoom": 5
```

Comment:
It looks like the data only starts appearing from zoom 10, so should we set this to 10 instead?